frontend/src/metabase/components/sortable/sortable.jsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+import styled from "styled-components";
+import {
+  SortableContainer as OriginalSortableContainer,
+  SortableElement,
+  SortableHandle,
+} from "react-sortable-hoc";
+import "./sortable.css";
+
+const SortableContainerWrapper = styled.div`
+  user-select: none;
+`;
+
+// Workaround for https://github.com/clauderic/react-sortable-hoc/issues/253
+// See issue demo here: https://github.com/metabase/metabase/issues/12870
+export function SortableContainer(WrappedComponent, config) {
+  const SortableList = OriginalSortableContainer(
+    props => (
+      <SortableContainerWrapper>
+        <WrappedComponent {...props} />
+      </SortableContainerWrapper>
+    ),
+    config,
+  );
+
+  // Makes sure the sortable-hoc always receives "react-sortable-hoc-helper" class
+  // The class just applies a big z-index to the dragged item created by sortable-hoc
+  // to prevent https://github.com/clauderic/react-sortable-hoc#item-disappearing-when-sorting--css-issues
+  function SortableListWithDraggedItemVisibilityFix({ helperClass, ...props }) {
+    return (
+      <SortableList
+        {...props}
+        helperClass={cx("react-sortable-hoc-helper", helperClass)}
```

Comment:
Do you like this approach of drilling classNames enough to add more cases?

Versus going with just styled components for it?

Or is this a situation where you don't want to deeply change existing things in the codebase?

Reply:
It's not a personal preference, but a part of `react-sortable-hoc's` API